<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="Desktop" failures="1" tests="4" name="Test" time="0.134" errors="0" timestamp="27 Feb 2021 20:19:41 GMT">
  <testcase classname="learningTestNG.LearningTestNG" name="test4" time="0.012"/>
  <testcase classname="learningTestNG.LearningTestNG" name="test6" time="0.011">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at learningTestNG.LearningTestNG.test6(LearningTestNG.java:58)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test6 -->
  <testcase classname="learningTestNG.LearningTestNG" name="test1" time="0.015"/>
  <testcase classname="learningTestNG.LearningTestNG" name="test3" time="0.003"/>
  <testcase classname="learningTestNG.LearningTestNG" name="test2" time="0.0">
    <ignored/>
  </testcase> <!-- test2 -->
  <testcase classname="learningTestNG.beforeAfter" name="@AfterSuite afterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuite -->
  <testcase classname="learningTestNG.beforeAfter" name="@BeforeTest beforeTest" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest beforeTest -->
  <testcase classname="learningTestNG.LearningTestNG" name="test5" time="0.0">
    <ignored/>
  </testcase> <!-- test5 -->
  <testcase classname="learningTestNG.beforeAfter" name="@AfterTest afterTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase classname="learningTestNG.beforeAfter" name="@BeforeSuite beforeSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeSuite -->
</testsuite> <!-- Test -->
